test/Makefile

```diff
@@ -263,7 +278,7 @@ endif
 agbot-prereq-build-anax: $(ANAX_SOURCE)/anax
 	@echo "Configuring agbot prerequisites"
 	if [[ "$(shell docker images -q $(DOCKER_ANAX_K8S_INAME) 2>/dev/null)" == "" ]]; then \
-		cd $(ANAX_SOURCE) && make anax-k8s-image; \
+		cd $(ANAX_SOURCE) && make anax-k8s-image && make anax-image; \
```

Comment:
I ran with sall pattern and MUL_AGENTS=3, but the anax image container was not built, so the agents didnt run. This is the first problem. The second problem is that make test did not fail when the extra agents failed to start.

Reply:
fixed.